src/System.Device.Gpio/CompatibilitySuppressions.xml

```diff
@@ -1,103 +1,103 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
   <Suppression>
-    <DiagnosticId>CP0002</DiagnosticId>
-    <Target>M:System.Device.Gpio.Drivers.RaspberryPi3Driver.AltMode.#ctor</Target>
+    <DiagnosticId>CP1003</DiagnosticId>
+    <Target>lib/net6.0-windows10.0.17763/System.Device.Gpio.dll</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0008</DiagnosticId>
+    <Target>T:System.Device.Spi.DataFlow</Target>
     <Left>lib/netstandard2.0/System.Device.Gpio.dll</Left>
     <Right>lib/net6.0-windows10.0.17763/System.Device.Gpio.dll</Right>
   </Suppression>
   <Suppression>
     <DiagnosticId>CP0002</DiagnosticId>
-    <Target>M:System.Device.Gpio.PinEventTypes.#ctor</Target>
+    <Target>M:System.Device.Spi.DataFlow.#ctor</Target>
```

Comment:
can you revert these? I don't think you've made any changes to System.Device.Gpio, this shouldn't be needed

Reply:
Sure, I must have missed this somewhere. This file keeps getting changed with the other one for no apparent reason (this PR doesn't even touch System.Device.Gpio)